plugins/homebrew/Brewfile.erb

```diff
@@ -5,8 +5,6 @@
 
 cask_args appdir: '/Applications'
 
-tap 'homebrew/cask'
```

Comment:
This fails and the error says it's not necessary. If we feel it still is we an add force it seems.
```
Tapping homebrew/cask
Error: Tapping homebrew/cask is no longer typically necessary.
Add --force if you are sure you need it for contributing to Homebrew.
Tapping homebrew/cask has failed!
```